scripts/create-clusters.sh

```diff
@@ -139,21 +139,14 @@ function fixup-cluster() {
   local i=${1} # cluster num
 
   local kubeconfig_path="$(kind get kubeconfig-path --name cluster${i})"
+  # Set container IP address as kube API endpoint in order for clusters to reach kube API servers in other clusters.
+  kind get kubeconfig --name "cluster${i}" --internal >${kubeconfig_path}
```

Comment:
Will this work for MacOS? The `--internal` flag according to the PR will use the internal container IP address. I believe MacOS relies on the use of `localhost`, which is what kind uses by default i.e. without `--internal` flag. However, I cannot verify this.

Reply:
@font, Ideally this should be taken care by kind, to implement a feature consistently across platform. Also theoretically the same problem would be present if `localhost` is used in `MacOS`. I too am not familiar with `MacOS`. Can somebody using `MacOS` confirm this please. @xunpan @aoxn @iamzhout can you please help in verifying this issue.

Reply:
Thanks @font for the clarification. I have updated the code. PTAL once again. Thanks!